eng/Signing.props

```diff
@@ -38,6 +36,7 @@
     <FileSignInfo Include="apphost.exe" CertificateName="None" />
     <FileSignInfo Include="comhost.dll" CertificateName="None" />
     <FileSignInfo Include="singlefilehost.exe" CertificateName="None" />
+    <FileSignInfo Include="Valleysoft.DockerCredsProvider.dll" CertificateName="None" />
```

Comment:
I was getting signing issues without ignoring this file. I'm not exactly sure how the current build operates without ignoring this file. I haven't changed any of the signing logic from `main`.

Reply:
I'm going to test not including this change. It might have been some kind of intermediary issue.

Reply:
Checked and yup, this change is definitely needed. In CI, I get:
```
D:\a\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\9.0.0-beta.24114.1\tools\Sign.proj(59,5): error SIGN004: Signing 3rd party library 'D:\a\_work\1\s\artifacts\tmp\Debug\ContainerSigning\1758\fil05473D37154D1C574CF9958F656E66DB/Valleysoft.DockerCredsProvider.dll' with Microsoft certificate 'MicrosoftDotNet500'. The library is considered 3rd party library due to its copyright: ''.
##[error].packages\microsoft.dotnet.arcade.sdk\9.0.0-beta.24114.1\tools\Sign.proj(59,5): error SIGN004: Signing 3rd party library 'D:\a\_work\1\s\artifacts\tmp\Debug\ContainerSigning\1758\fil05473D37154D1C574CF9958F656E66DB/Valleysoft.DockerCredsProvider.dll' with Microsoft certificate 'MicrosoftDotNet500'. The library is considered 3rd party library due to its copyright: ''.
```

Reply:
I _think_ we already get it signed from https://github.com/dotnet/sdk/blob/ed857658103e3b2b65920df236ae26cf7c66ec89/eng/Signing.props#L54

but that also means we should probably sign it with the `ExternalCertificateId` here

Reply:
I'll give that a try.

Reply:
That worked. So, I'll keep it with using `ExternalCertificateId`.